config.js

```diff
@@ -14,34 +15,35 @@ module.exports = {
   },
   useMocks: useMocks,
   dateTimeFormat: 'DD-MM-YYYY, hh:mma',
-  port: process.env.PORT || 8080,
+  port: port,
   session: {
-    secret: process.env.SESSION_SECRET || 'howdoesyourgardengrow',
-    ttl: process.env.SESSION_TTL || 1800, /* 30 mins timeout */
-    name: process.env.SESSION_KEY || 'rotm.sid'
+    secret: process.env.SESSION_SECRET,
+    ttl: process.env.SESSION_TTL, /* 30 mins timeout */
```

Comment:
can you remove the comment as that was referring to the old default

Reply:
will do :)